frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-community-provider-modal.tsx

```diff
@@ -9,81 +9,73 @@ import {
 
-  onIgnoreChange = (checked) => {
-    this.setState({ ignoreWarnings: checked });
-  };
+  const onIgnoreChange = React.useCallback((checked) => {
+    setIgnoreWarnings(checked);
+  }, []);
 
-  submit = (event) => {
-    event.preventDefault();
-    this.props.showCommunityOperators(this.state.ignoreWarnings);
-    this.props.close();
-  };
+  const submit = React.useCallback(
+    (event) => {
+      event.preventDefault();
+      showCommunityOperators(ignoreWarnings);
+      close();
+    },
+    [close, ignoreWarnings, showCommunityOperators],
+  );
 
-  render() {
-    const { ignoreWarnings } = this.state;
-    const submitButtonContent = <>Continue</>;
-    return (
-      <form onSubmit={this.submit} className="modal-content co-modal-ignore-warning">
-        <ModalTitle>Show Community Operator</ModalTitle>
-        <ModalBody>
-          <div className="co-modal-ignore-warning__content">
-            <div className="co-modal-ignore-warning__icon">
-              <InfoCircleIcon />
-            </div>
-            <div>
-              <p>
-                Community Operators are operators which have not been vetted or verified by Red Hat.
-                Community Operators should be used with caution because their stability is unknown.
-                Red Hat provides no support for Community Operators.
-                {RH_OPERATOR_SUPPORT_POLICY_LINK && (
-                  <span className="co-modal-ignore-warning__link">
-                    <ExternalLink
-                      href={RH_OPERATOR_SUPPORT_POLICY_LINK}
-                      text="Learn more about Red Hat’s third party software support policy"
-                    />
-                  </span>
-                )}
-              </p>
-              <Checkbox
-                className="co-modal-ignore-warning__checkbox"
-                onChange={this.onIgnoreChange}
-                isChecked={ignoreWarnings}
-                id="do-not-show-warning"
-                label="Do not show this warning again"
-              />
-            </div>
+  return (
+    <form onSubmit={submit} className="modal-content co-modal-ignore-warning">
+      <ModalTitle>{t('olm~Show community Operator')}</ModalTitle>
+      <ModalBody>
+        <div className="co-modal-ignore-warning__content">
+          <div className="co-modal-ignore-warning__icon">
+            <InfoCircleIcon />
           </div>
-        </ModalBody>
-        <ModalSubmitFooter
-          submitText={submitButtonContent}
-          inProgress={false}
-          errorMessage=""
-          cancel={this.props.close}
-        />
-      </form>
-    );
-  }
-}
+          <div>
+            <p>
+              {t(
+                'olm~Community Operators are Operators which have not been vetted or verified by Red Hat. Community Operators should be used with caution because their stability is unknown. Red Hat provides no support for Community Operators.',
```

Comment:
Here you have "Community Operators," but you have "community Operators" in other places.

```suggestion
                'olm~Community Operators are Operators which have not been vetted or verified by Red Hat. Community Operators should be used with caution because their stability is unknown. Red Hat provides no support for community Operators.',
```